.github/workflows/build.yml

```diff
@@ -2,6 +2,8 @@ name: build
 
 on:
   push:
+    paths-ignore:
+      - '**.md'
```

Comment:
we don't normally push to those branches so no point in ignoring here - maybe you wanted to update the `pull_request` conf?

Reply:
Ahh.. he who can read and all.. Thanks!